integration/grpc-web/main.go

```diff
@@ -29,49 +25,50 @@ var (
 func main() {
 	flag.Parse()
 
-	port := 9090
-	if *enableTls {
-		port = 9091
+	lis, err := net.Listen("tcp", "localhost:9090")
+	if err != nil {
+		log.Fatalf("failed to listen: %v", err)
 	}
+	log.Println("create server at localhost:9090")
+	server := grpc.NewServer()
+	rpx.RegisterDashStateServer(server, &stateService{})
+	if err := server.Serve(lis); err != nil {
+		log.Fatalf("failed to serve: %v", err)
+	}
+}
 
-	grpcServer := grpc.NewServer()
-
-	rpx.RegisterDashStateServer(grpcServer, &stateService{})
-	rpx.RegisterDashAPICredsServer(grpcServer, &credsService{})
-
-	grpclog.SetLogger(log.New(os.Stdout, "exampleserver: ", log.LstdFlags))
+type stateService struct{}
 
-	wrappedServer := grpcweb.WrapServer(grpcServer)
```

Comment:
@PhilipMantrov I'm confused about this, why did you remove this? Doesn't this mean the go server no longer speaks grpc-web?

Reply:
Don't remember why I deleted this, sorry.
I think we need return this part.

Reply:
Cool, np, I'll work on adding it back. Give me an excuse to re-learn some go. :-)